conans/test/conftest.py

```diff
@@ -102,6 +102,11 @@
         "default": "system",
         "system": {"path": {'Windows': 'C:/bazel/bin',
                             "Darwin": '/Users/jenkins/bin'}},
+    },
+    "intel_oneapi": {
```

Comment:
If this is NOT installed in CI, you need to put it in you ``conftest_user.py`` to try things (it is good if you put it in this file as a comment, to explain it, but not activate it and then ``xfail`` the test)